<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="IGT_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>751</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>In Game Time:</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>465</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <layout class="QGridLayout" name="missile_layout">
       <item row="0" column="0">
        <widget class="QLabel" name="item_00">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Missile</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="item_13">
       <property name="text">
        <string>Spider</string>
       </property>
      </widget>
     </item>
     <item row="3" column="10">
      <layout class="QGridLayout" name="artifact_layout">
       <item row="0" column="0">
        <widget class="QLabel" name="item_35">
         <property name="text">
          <string>Artifacts</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="item_12">
       <property name="text">
        <string>Boost</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="item_10">
       <property name="text">
        <string>Morph</string>
       </property>
      </widget>
     </item>
     <item row="2" column="10">
      <widget class="QLabel" name="item_25">
       <property name="text">
        <string>XRay</string>
       </property>
      </widget>
     </item>
     <item row="3" column="8">
      <widget class="QLabel" name="item_34">
       <property name="text">
        <string>Grapple</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="item_02">
       <property name="text">
        <string>Wave</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLabel" name="item_32">
       <property name="text">
        <string>Phazon</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <layout class="QGridLayout" name="etank_layout">
       <item row="0" column="0">
        <widget class="QLabel" name="item_01">
         <property name="text">
          <string>Etank</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="item_30">
       <property name="text">
        <string>Varia</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QLabel" name="item_04">
       <property name="text">
        <string>Plasma</string>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <widget class="QLabel" name="item_05">
       <property name="text">
        <string>Scan</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="item_31">
       <property name="text">
        <string>Gravity</string>
       </property>
      </widget>
     </item>
     <item row="1" column="10">
      <widget class="QLabel" name="item_15">
       <property name="text">
        <string>Thermal</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="item_03">
       <property name="text">
        <string>Ice</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="item_11">
       <property name="text">
        <string>Bombs</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <layout class="QGridLayout" name="pb_layout">
       <item row="0" column="0">
        <widget class="QLabel" name="item_14">
         <property name="text">
          <string>PB</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="8">
      <widget class="QLabel" name="item_23">
       <property name="text">
        <string>Flamethr</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="item_22">
       <property name="text">
        <string>Ice Sprea</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="item_21">
       <property name="text">
        <string>WaveBus</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="item_20">
       <property name="text">
        <string>Supers</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="item_33">
       <property name="text">
        <string>Charge</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QLabel" name="item_24">
       <property name="text">
        <string>SJ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="retry_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>420</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Retry</string>
    </property>
   </widget>
   <widget class="QLabel" name="dolphin_status">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dolphin status: Connected</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>532</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
